### PR TITLE
rebranded strings (koinu)

### PR DESCRIPTION
**Pull request by daank-c**

### CODE REVIEW

**Comment by patricklodder** [comment]
The translations get updated with a script because the translation needs to be marked too (painful to do manually), so all the files under `src/qt/locale` need to be ignored.

**Comment by patricklodder** [comment]
> Should the translations themselves be changed though?

Not by you. When we have a fix on the release we will run the script and invite translators.

File: src/rpc/mining.cpp
Comment (patricklodder) [inline src/rpc/mining.cpp:3]: If you *really* want to change the copyright line, please use a range when you edit it. In this case it should be 2021-2023.

PS: I have a [script](https://gist.github.com/patricklodder/210a449896ece9ca4a8f872328198e3f) for that too, so you don't have to bother really - sorry.

Comment (patricklodder) [reply src/rpc/mining.cpp:3]: No need to close because you can make a single commit from all these. That's a good skill to learn.

See https://github.com/dogecoin/dogecoin/blob/master/CONTRIBUTING.md#squashing-commits

**Comment by philipperemy** [comment]
Not a big deal but should not Koinu be referenced with a capital K?

**Comment by patricklodder** [comment]
> Not a big deal but should not Koinu be referenced with a capital K?

It's used like "cent", so I'd keep it lower case in most languages, except when the language has a rule to capitalize the K, like German. This is also what we've done historically.

**Review by michilumin — CHANGES_REQUESTED** [review]
Honestly, this is such a simple and straightforward change to make, I'm willing to approve if you do back out the manual copyright change, keep it lowercase, and squash your commits (as is Pat's preferred style) - but core saying "satoshis" has been a nit of mine for a while.

**Comment by patricklodder** [comment]
> as is Pat's preferred style

Although I do agree that clean commits are good practice, I have had no hand in [the process around rebasing and squashing](https://github.com/dogecoin/dogecoin/blob/master/CONTRIBUTING.md#squashing-commits) other than fixing the last sentence textually; I'm merely following the process we agreed upon.

If you want something different, please propose changes.

**Comment by michilumin** [comment]
> > as is Pat's preferred style
> 
> Although I do agree that clean commits are good practice, I have had no hand in [the process around rebasing and squashing](https://github.com/dogecoin/dogecoin/blob/master/CONTRIBUTING.md#squashing-commits) other than fixing the last sentence textually; I'm merely following the process we agreed upon.
> 
> If you want something different, please propose changes.

`If your pull request is accepted for merging, you may be asked by a maintainer to squash ...`

What change?  That says it's based on maintainer preference. "You may be asked..."  where "You must, to proceed" would actually be a change. You asked him to, so I included it in my approval criteria. So not sure why you're taking exception. It's being done exactly as it says on the tin.

This doesn't even really matter so much at this point since github allows us to squash commits on merge. If you think it'd be a good skill for him to learn - agreed, and cool - but it doesn't really have anything to do with approval of this.

**Comment by philipperemy** [comment]
The one who performs the PR merge can choose to squeeze the commits. Github can easily handle that yeah.

**Comment by patricklodder** [comment]
> squash commits on merge 

I'm sure everyone is aware that GH squash & merge unfortunately does not (because it cannot) retain the commit signature of the author but instead replaces it with GitHub's signature. The option is disabled on the repo for this reason - replacing commit signatures at maintainer discretion isn't cool. We can discuss this again... I don't mind.

None of this is about my preferences though; the text is a one-on-one copy from Bitcoin Core.

**Review by patricklodder — APPROVED** [review]
ACK. Tested on Ubuntu Jammy x86_64 (rpc help) and macOS Ventura arm64 (dogecoin-qt)

looks good:
Screenshot

**Review by michilumin — APPROVED** [review]
ACK - LGTM.

**Pull request merged**
